packages/mobx-state-tree/src/utils.ts

```diff
@@ -176,6 +176,11 @@ export function argsToArray(args: IArguments): any[] {
     return res
 }
 
+export function invalidateComputed(target: any, propName: string) {
+    const atom = getAtom(target, propName) as any
+    atom.trackAndCompute()
```

Comment:
I think this would ignore normal mobx transactions? Would there be potential risks in that?

Reply:
To be honest, this part is terra-incognita for me. As I wrote before "```observableInstanceCreated``` is not ```@observable```, but we must invalidate ```@computed``` ```snapshot``` after initializetion".
I did not find any documented way to do so and was stuck around this problem for a while. 
I ended up exploring mobx TS type definitions for something like ```myComputed.forceRecompute()``` and found such solution (almost by accident).
I used it to invalidate ```snapshot``` at this branch and ```path``` at [#872](https://github.com/mobxjs/mobx-state-tree/pull/872) successfully. 
If you give me a clue about what to check, I can explore further ;)